serverless/verify-backup/src/server.ts

```diff
@@ -0,0 +1,16 @@
+import express, {Request, Response} from 'express'
+import bodyParser from 'body-parser'
+
+const app = express();
+const PORT = process.env.PORT || 8080;
+
+app.use(bodyParser.json());
+
+app.post('/', (req: Request, res:Response) => {
+  console.log('Incoming pubsub message', JSON.stringify(req.body))
```

Comment:
is this supposed to trigger something? didn't test because im not sure what it's supposed to do

Reply:
As discussed, this is to accept the incoming POST request from the cloud scheduler which pings the cloud run instance, this in turn wakes up the instance and executes the entrypoint script. I'll elaborate on this in the PR description.